### PR TITLE
Popup branch

### PR DESCRIPTION
**Pull request by Metsanpeitto**
# Portfolio: details popup window

- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Understand how to parse a Figma design to create a UI.
- Use objects to store and access data.

### CODE REVIEW

File: scss/layout/_modal.scss [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+                font-style: normal;
+                font-weight: 500;
+                font-size: 12px;
+                line-height: 16px;
+                letter-spacing: 0.03em;
+                color: $blue-4;
+                margin: unset;
+              }
+            }
+          }
```
Comment (sinansevgi) [inline scss/layout/_modal.scss:296]: - [ ] This section has a responsiveness issue.

- In the mobile resolution (~380px), It doesn't fit its parent container and creates a vertical scroll bar.
Mobile

- In the desktop resolution(~1100px), It has non-visible parts.
Desktop

I tested the project with firefox and chrome browsers. These issues occur on both of them. 

Suggestion: You can use [minmax](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax()) CSS function to for giving minimum sizes to your grid regions.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Metsanpeitto and @ntzwilly  , 

Excellent work with your javascript :+1:, 
You well-abstracted application logic and the dom-manipulation in your application. :clap: 
The projects section and modals dynamically generated with the javascript as expected. :star: 

I noticed only one small issue with the responsiveness of the modals. Please check the inline comment that I left. 

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scss/layout/_modal.scss (hunk 1 above)
Comment (Metsanpeitto) [reply scss/layout/_modal.scss:296]: Thanks for pointing it out. I totally missed that when testing on my ends.
I have already fixed it and also added flex and flex-wrap to fix the 'technology bubbles' being chopped when the screen gets to the range between 320px and 380px.
Your critique was instructive and your suggestion helpful.
Thanks.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**